redesign/src/styles/layout/_nav.scss

```diff
@@ -0,0 +1,165 @@
+    position: relative;
+
+    @media screen and (min-width: $show-nav) {
+        background-color: $red;
+        height: 100%;
+    }
+}
+
+.menu {
+    &--button {
+        background: $red;
+        color: #FFFFFF;
+        height: 114px;
+        position: fixed;
+        right: 0;
+        top: 0;
+        width: 104px;
+
+        &[aria-expanded="true"] {
+            background: $yellow;
+            transition: background 0.3s ease-in-out;
+        }
+
+        &[aria-expanded="true"] {
+            .menu-icon--line {
+                $yValue: 18px;
+                $deg: 45deg;
+
+                &:nth-child(1) {
+                    transform: translateY($yValue) rotate($deg);
+                }
+
+                &:nth-child(2) {
+                    opacity: 0;
+                }
+
+                &:nth-child(3) {
+                    transform: translateY(-$yValue) rotate(-$deg);
+                }
+            }
+        }
+
+        @media screen and (min-width: $show-nav) {
+            display: none;
+        }
+    }
+
+    &--list {
+        @media screen and (max-width: ($show-nav - 1px)) {
+            align-items: flex-end;
+            flex-direction: column;
+            height: 0;
+            padding: 0;
+            position: fixed;
+            right: 0;
+            top: 114px;
+            width: 320px;
+
+            &[data-visible="true"] {
+                animation: expand 0.3s ease-in-out forwards;
+                display: flex;
+            }
+
+            @media screen and (max-width: ($show-nav - 1px)) {
+                background-color: white;
+            }
+
+            &:active {
+                background: white;
+            }
+        }
+
+        @media screen and (min-width: $show-nav) {
+            display: flex !important;
+            flex-direction: column;
+            height: 452px;
+            width: 300px;
+            @include padding(20px, 0, 90px, 0);
+
+        }
+    }
+
+    &--text {
+        background: $red;
+        display: flex;
+        flex: 1 0 auto;
+        margin-bottom: 1px;
+        width: 320px;
+        height: 75px;
+
+        @media screen and (max-width: ($show-nav - 1px)) {
+            justify-content: left;
+        }
+
+        >a {
+            width: 320px;
+            transition: background 0.3s ease-in-out;
+
+            &:hover {
+                background: $darkred;
+                color: #FFFFFF;
+            }
+
+            @media screen and (max-width: ($show-nav - 1px)) {
+                font-size: .9rem;
+            }
+        }
+    }
+
+    @keyframes expand {
+        from {
+            height: 0;
+            opacity: 0;
+        }
+
+        to {
+            height: 375px;
+            opacity: 1;
+        }
+    }
+}
+
+.menu-icon {
+    &--line {
+        background-color: white;
+        height: 3px;
+        margin: 10px auto 15px;
+        transition: all 0.3s ease-in-out;
+        width: 40px;
+    }
+}
+
+a.link {
```

Comment:
These are pretty vague classnames and kind of divorced from BEM. Maybe `menu__link`?